nano/node/common.hpp

```diff
@@ -104,6 +91,19 @@ struct ip_address_hash<4>
 		return result;
 	}
 };
+}
+
+namespace std
```

Comment:
As far as I can tell, this remains in the namespace `::std`

Reply:
The diff itself is not very clear about what has moved into what namespace, might be easier to view the whole file instead. There are only certain things which are allowed to be added to the std namespace, std::hash specializations being one of them.

Reply:
Got it, thanks !